### PR TITLE
Adjust user profile score to closer match web

### LINKED ISSUE #7845: Some text on a user profile doesnt align correctly
**Screenshots or videos showing encountered issue:** 
lazer:
VVfHBEaUms

web:
6Ru87e6FpD

**osu!lazer version:** 
commit de6ea60ae1c192f360989321a7e2cef90541b4d9

**Used user profile:** https://osu.ppy.sh/users/2123060

### PR DESCRIPTION
**Pull request by recapitalverb**
Fixes #7845
Screenshot_20200215_165631

### CODE REVIEW

**Comment by bdach** [comment]
I modified the existing visual test to include the original case - no assertions, but a least visually it'll look off if it regresses again. Looks fine otherwise.

**Comment by Joehuu** [comment]
This will add extra space on first place ranks.

Lazer (this PR):
image

Web:
image

**Comment by bdach** [comment]
Resolved:

image

I've also adjusted sizings and drawable hierarchy to closer match web implementation.

File: osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs [hunk 1]
```diff
@@ -22,10 +21,9 @@ namespace osu.Game.Overlays.Profile.Sections.Ranks
     public class DrawableProfileScore : CompositeDrawable
     {
         private const int height = 40;
-        private const int performance_width = 80;
+        private const int performance_width = 100;
 
         private const float performance_background_shear = 0.45f;
-        private static readonly float performance_background_width = performance_width + (height / 4f * MathF.Tan(performance_background_shear));
```
Comment (recapitalverb) [inline osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs]: This shouldn't be removed. See #7710.

Comment (bdach) [reply osu.Game/Overlays/Profile/Sections/Ranks/DrawableProfileScore.cs]: It didn't need to be there in the first place, if we're going for absolute sizing parity with web.

| this PR | web |
| :-: | :-: |
| image | image |

Note the consistent 100 width in both places. The previous solution wasn't really necessary and I should have paid better attention during review. Going forward I will be checking element dimensions against web very strictly.

**Pull request merged**